local/local.go

```diff
@@ -494,6 +501,116 @@ func (i *Image) Save(additionalNames ...string) error {
 	return nil
 }
 
+func (i *Image) SaveFile() (string, error) {
+	f, err := os.CreateTemp("", "imgutil.local.image.export.*.tar")
```

Comment:
```suggestion
	if i.inspect.Os == "windows" {
		return "", errors.New("not yet implemented")
	}
	f, err := os.CreateTemp("", "imgutil.local.image.export.*.tar")
```

Reply:
WDYT @flix- @jabrown85? Do we need this to work for Windows images?

Reply:
It does _technically_ work on my windows machine. Do we have a concept of an unsupported feature? It would be nice to allow it to work if it can and skip the tests instead as a way of saying "unsupported"

Reply:
Managed to get a Windows machine... I assume it is working on your Windows clients because you are using Linux containers (right click on the Docker desktop tray icon and you'll see 'switch to Windows containers...'). IMHO Windows Server uses Windows containers per default. For Linux containers everything works as expected (osType is 'linux' and none of the Windows specifics apply). However for Windows containers I can reproduce the exact same error as reported by Github actions. I will need some time to investigate further. Current assumption is that the Windows base layers are the problematic part. If we have a Windows image they are added by default when using newImage(). I am reusing a layer here which also includes those base directories/files for windows containers (and a lot more than the ones generated by newImage()) and the image is imported without an error but is still not present afterwards. Manually removing the Windows base layer written by newImage() and only using the ones from the re-used layers works without any issues...

Reply:
oh, doh! I did forget to change that. I think I assumed the tests would automate that bit for me. You can tell how often I use windows for container work 😬